src/en_restraint.F90

```
@@ -161,7 +175,7 @@ subroutine energy_restraint(x, y, z, px, py, pz, eclas)
    end subroutine energy_restraint
 
    subroutine en_rest_finalize()
-      deallocate (fxr, fyr, fzr)
+      if (allocated(fxr)) deallocate (fxr, fyr, fzr)
```

@suchanj just FYI in finalization routines one should always check whether things are allocated before deallocating them since ABIN can fail early before the inititialization routine is called.